programs/ssl/ssl_client2.c

```diff
@@ -3177,6 +3220,12 @@ int main( int argc, char *argv[] )
 
         mbedtls_printf( " ok\n" );
 
+#if defined(MBEDTLS_SSL_EARLY_DATA) && defined(MBEDTLS_SSL_CLI_C)
+        /* TODO: We can log the actual early data status when reconnect
```

Comment:
Not clearly, why it is needed? if it is not scope of this PR. Please remove it.

Reply:
ditto, Will remove it.